<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mer::Internal::MerEmulatorDeviceWidget</class>
 <widget class="QWidget" name="Mer::Internal::MerEmulatorDeviceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::ExpandingFieldsGrow</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="emulatorLabel">
     <property name="text">
      <string>Virtual Machine:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="emulatorVmLabelEdit">
     <property name="text">
      <string>none</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="factorySnapshotLabel">
     <property name="text">
      <string>Fa&amp;ctory snapshot:</string>
     </property>
     <property name="buddy">
      <cstring>factorySnapshotToolButton</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="hostLabel">
     <property name="text">
      <string>Host:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="hostLabelEdit">
     <property name="text">
      <string>localhost</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="macLabel">
     <property name="text">
      <string>MAC:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="macLabelEdit">
     <property name="text">
      <string>none</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="sshPortLabel">
     <property name="text">
      <string>SSH port:</string>
     </property>
     <property name="buddy">
      <cstring>sshPortSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QSpinBox" name="sshPortSpinBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimum">
        <number>1024</number>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="value">
        <number>2223</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="sshPortInfoLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="configFolderLabel">
     <property name="text">
      <string>&quot;config&quot; folder:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="configFolderLabelEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>none</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="sshFolderLabel">
     <property name="text">
      <string>&quot;ssh&quot; folder:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="sshFolderLabelEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>none</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="userNameLabel">
     <property name="text">
      <string>&amp;Username:</string>
     </property>
     <property name="buddy">
      <cstring>userLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="userLineEdit">
     <property name="text">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="sshKeyLabel">
     <property name="text">
      <string>SSH key:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLabel" name="sshKeyLabelEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>none</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="connectionTimeoutLabel">
     <property name="text">
      <string>&amp;Timeout:</string>
     </property>
     <property name="buddy">
      <cstring>timeoutSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QSpinBox" name="timeoutSpinBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="suffix">
      <string>s</string>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>1000</number>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="freePortsLabel">
     <property name="text">
      <string>F&amp;ree ports:</string>
     </property>
     <property name="buddy">
      <cstring>portsLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="portsLineEdit">
       <property name="toolTip">
        <string>You can enter lists and ranges like this: 1024,1026-1028,1030</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="portsWarningLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="qmlLivePortsLabel">
     <property name="text">
      <string>&amp;QmlLive ports:</string>
     </property>
     <property name="buddy">
      <cstring>qmlLivePortsLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="qmlLivePortsLineEdit">
       <property name="toolTip">
        <string>You can enter lists and ranges like this: 1024,1026-1028,1030</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="qmlLivePortsInfoLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="qmlLivePortsWarningLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLineEdit" name="factorySnapshotLineEdit">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The name of the virtual machine snapshot that is used to restore the emulator to the &lt;span style=&quot; font-style:italic;&quot;&gt;factory&lt;/span&gt; state.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="factorySnapshotToolButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>userLineEdit</zorder>
  <zorder>userNameLabel</zorder>
  <zorder>connectionTimeoutLabel</zorder>
  <zorder>timeoutSpinBox</zorder>
  <zorder>freePortsLabel</zorder>
  <zorder>emulatorVmLabelEdit</zorder>
  <zorder>emulatorLabel</zorder>
  <zorder>sshKeyLabel</zorder>
  <zorder>sshKeyLabelEdit</zorder>
  <zorder>configFolderLabel</zorder>
  <zorder>sshFolderLabel</zorder>
  <zorder>configFolderLabelEdit</zorder>
  <zorder>sshFolderLabelEdit</zorder>
  <zorder>sshPortLabel</zorder>
  <zorder>hostLabel</zorder>
  <zorder>hostLabelEdit</zorder>
  <zorder>macLabel</zorder>
  <zorder>macLabelEdit</zorder>
  <zorder></zorder>
  <zorder>qmlLivePortsLabel</zorder>
  <zorder>factorySnapshotLabel</zorder>
  <zorder>horizontalLayoutWidget</zorder>
 </widget>
 <tabstops>
  <tabstop>factorySnapshotLineEdit</tabstop>
  <tabstop>factorySnapshotToolButton</tabstop>
  <tabstop>sshPortSpinBox</tabstop>
  <tabstop>userLineEdit</tabstop>
  <tabstop>timeoutSpinBox</tabstop>
  <tabstop>portsLineEdit</tabstop>
  <tabstop>qmlLivePortsLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
